packages/core/src/indexing/__tests__/build-indexing.test.ts

```diff
@@ -1,17 +1,30 @@
+import { jest } from '@jest/globals'
 import tmp from 'tmp-promise'
 import { buildIndexing, UnsupportedDatabaseProtocolError } from '../build-indexing.js'
 import { PostgresIndexApi } from '../postgres/postgres-index-api.js'
 import { SqliteIndexApi } from '../sqlite/sqlite-index-api.js'
-import { LoggerProvider } from '@ceramicnetwork/common'
+import { LoggerProvider, Networks } from '@ceramicnetwork/common'
 import pgTest from '@databases/pg-test'
+import pgSetup from '@databases/pg-test/jest/globalSetup'
+import pgTeardown from '@databases/pg-test/jest/globalTeardown'
 
 const diagnosticsLogger = new LoggerProvider().getDiagnosticsLogger()
 // @ts-ignore default import
 const getDatabase = pgTest.default
 
+jest.setTimeout(150000)
+
 describe('sqlite', () => {
   let databaseFolder: tmp.DirectoryResult
 
+  beforeAll(async () => {
```

Comment:
Why do we need to setup Postgres when running the SQLite tests please?
does this only affect the test setup, or does it become necessary to run Ceramic even with historical indexing disabled?

Reply:
oooo good catch! You only need postgres if using postgres =)